src/components/SplitVertical/SplitVertical.spec.jsx

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import assert from 'assert';
+import sinon from 'sinon';
+import describeWithDOM from '../../util/describe-with-dom';
+import _ from 'lodash';
+import { common } from '../../util/generic-tests';
+import SplitVertical from './SplitVertical';
+import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
+
+describe('SplitVertical', () => {
+	common(SplitVertical);
+
+	describe('render', () => {
+		it('should render an inner element with the left, right, and divider elements as children', () => {
+			const wrapper = shallow(
+				<SplitVertical />
+			);
+
+			assert.equal(wrapper.find('.lucid-SplitVertical').length, 1);
+			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner').length, 1);
+			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-LeftPane').length, 1);
+			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-Divider').length, 1);
+			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-RightPane').length, 1);
+		});
+	});
+
+	describe('props', () => {
+		describe('isExpanded', () => {
+			it('should default to true', () => {
+				const wrapper = shallow(
+					<SplitVertical />
+				);
+
+				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 1);
+			});
+
+			it('should apply the &-is-expanded css class when true', () => {
+				const wrapper = shallow(
+					<SplitVertical isExpanded={true} />
+				);
+
+				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 1);
+			});
+
+			it('should apply the &-is-expanded css class when false', () => {
+				const wrapper = shallow(
+					<SplitVertical isExpanded={false} />
+				);
+
+				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 0);
+			});
+		});
+
+		describeWithDOM('isAnimated', () => {
+			let wrapper;
+
+			afterEach(() => {
+				if (wrapper) {
+					wrapper.unmount();
+				}
+			});
+
+			it('should default to false', (done) => {
+				wrapper = mount(
+					<SplitVertical />
+				);
+
+				_.delay(() => {
+					assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-animated').length, 0);
+					done();
+				}, 10)
+			});
+
+			it('should apply the &-is-animated class when true, after initial render', (done) => {
+				wrapper = mount(
+					<SplitVertical isAnimated={true} />
+				);
+
+				_.delay(() => {
+					assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-animated').length, 1);
+					done();
+				}, 10)
+			});
+
+			it('should apply the &-is-animated class when false', (done) => {
+				wrapper = mount(
+					<SplitVertical isAnimated={false} />
+				);
+
+				_.delay(() => {
```

Comment:
I feel like we should somehow tag these kinds of tests as potentially "flaky". I wrote a few of these for `TextInput` as well and they've failed on rare occasions due to timing issues.